sysl2/sysl/syslutil/chroot_fs.go

```diff
@@ -70,3 +79,11 @@ func (fs *ChrootFs) Chmod(name string, mode os.FileMode) error {
 func (fs *ChrootFs) Chtimes(name string, atime time.Time, mtime time.Time) error {
 	return fs.fs.Chtimes(fs.join(name), atime, mtime)
 }
+
+func (fs *ChrootFs) openAllowed(name string) (err error) {
+	modulePath := filepath.Join(fs.root, name)
```

Comment:
will this work with the following dir structure: `/a/b/c/d/e` rooted at `/` module: `/a/b/c/d/e/foo.sysl` attempting to import `../../bar.sysl`?

I think you need to do a few calls to filepath.EvalSymLinks and filepath.Clean

Reply:
a lot of filepath functions already call `Clean` automatically, but will add the EvalSymLinks in next commit